framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/OpenShiftOperatorScenario.java

```diff
@@ -108,6 +118,25 @@ private void createRoleBindingsInProject(Project project) {
         project.getOpenShiftAdmin().rbac().roleBindings().inNamespace(project.getName()).create(roleBinding);
     }
 
+    private void createClusterRoleInProject(Project project) {
+        logger.info("Creating cluster role in project '" + project.getName() + "' from " + OpenShiftResource.CLUSTER_ROLE.getResourceUrl().toString());
+        ClusterRole role = project.getOpenShiftAdmin().rbac().clusterRoles().load(OpenShiftResource.CLUSTER_ROLE.getResourceUrl()).get();
+        project.getOpenShiftAdmin().rbac().clusterRoles().inNamespace(project.getName()).createOrReplace(role);
+    }
+
+    private void createClusterRoleBindingsInProject(Project project) {
+        logger.info("Creating cluster role bindings in project '" + project.getName() + "' from " + OpenShiftResource.CLUSTER_ROLE_BINDING.getResourceUrl().toString());
+        ClusterRoleBinding roleBinding = project.getOpenShiftAdmin().rbac().clusterRoleBindings().load(OpenShiftResource.CLUSTER_ROLE_BINDING.getResourceUrl()).get();
+        roleBinding.getMetadata().setName(OPERATOR_DEPLOYMENT_NAME + project.getName());
```

Comment:
This is to avoid collisions with other tests. 
The problem with this is that we need to remove the object when we delete the project (see the undeploy method above) as I confirmed that this is not done by Openshift when we delete the project.